integration-libs/cdc/root/service/cdc-js.service.spec.ts

```diff
@@ -318,47 +318,51 @@ describe('CdcJsService', () => {
     });
     it('should call register', () => {
       spyOn(winRef.nativeWindow['gigya'].accounts, 'initRegistration');
-      service.registerUserWithoutScreenSet({
-        uid: 'uid',
-        password: 'password',
-      });
-      expect(
-        winRef.nativeWindow['gigya'].accounts.initRegistration
-      ).toHaveBeenCalled();
+      expect(service.registerUserWithoutScreenSet).toBeTruthy();
+      service
+        .registerUserWithoutScreenSet({
+          uid: 'uid',
+          password: 'password',
+        })
+        .subscribe(() => {
+          expect(
+            winRef.nativeWindow['gigya'].accounts.initRegistration
+          ).toHaveBeenCalled();
+        });
```

Comment:
[MINOR]
It would be good call [`done()` function of jasmine](https://jasmine.github.io/tutorials/async#callbacks), after the `expect()`, inside the subscription's callback.
If we didn't put `done()` here, when the method doesn't emit anything, 
the test would pass. Which would false positive result. 

Similar remark applies also to other tests in this PR

Reply:
Will be handled in https://github.com/SAP/spartacus/pull/16169